tests/test_models.py

```diff
@@ -1034,6 +1034,21 @@ def test_get_course_and_course_run_no_content_items(self, contains_courses_mock)
         enterprise_customer_catalog = factories.EnterpriseCustomerCatalogFactory()
         assert enterprise_customer_catalog.get_course_and_course_run('fake-course-run-id') == (None, None)
 
+    def test_title_length(self):
+        """
+        Test `EnterpriseCustomerCatalog.title` field can take 255 characters.
+        """
+        faker = FakerFactory.create()
+        uuid = faker.uuid4()  # pylint: disable=no-member
+        title = 't'*255
```

Comment:
nit: spaces around arithmetic character `'t' * 255`

Reply:
would be better to use faker for this too. something like
```
faker.text(max_nb_chars=255)
```